itemManagement/views.py

```diff
@@ -18,7 +18,7 @@ class HomePage(SearchView):
 
     def get_queryset(self):
         queryset = super(HomePage, self).get_queryset()
-        return queryset.filter(deleted=False)
+        return queryset.filter(deleted=False).order_by('-lastUsed')
```

Comment:
This is not what we want. What you are doing here is ordering the already ordered results of a successful search. We want a default list, I already have a todo where it is suppose to go

Reply:
ISSUE STILL HERE